core/rfb.js

```diff
@@ -11,12 +11,14 @@ import { toUnsigned32bit, toSigned32bit } from './util/int.js';
 import * as Log from './util/logging.js';
 import { encodeUTF8, decodeUTF8 } from './util/strings.js';
 import { dragThreshold } from './util/browser.js';
+import { clientToElement } from './util/element.js';
```

Comment:
To me this seems to fit in `util/browser.js`, what's the reasoning here?

Reply:
`browser.js` is about browser capabilities